### PR TITLE
Contact page

### PR DESCRIPTION
**Pull request by balqees0325**
We ensure that all linter rules are followed.
contact page was added to HTML File,
CSS styles were added to the contact page by pair coding partners.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -371,5 +371,61 @@ <h2>About Myself</h2>
       </div>
     </section>
     <!-- end of about -->
+
+    <!-- contact page with pairs -->
```
Comment (rloterh) [inline index.html]: - [x] **OPTIONAL**
You left comments in your code hosted on the repository. To ensure cleaner code on your repository kindly remove all comments in your code.

File: index.html [hunk 2]
```diff
@@ -371,5 +371,61 @@ <h2>About Myself</h2>
...
+        <h2>Contact me</h2>
+        <p>
+          If you have an application you are
+          interested in developing, a feature that
+          you need built or a project that needs
+          coding. I’d love to help with it love
+          what you see.
+        </p>
+      </div>
+      <form>
```
Comment (rloterh) [inline index.html]: - [x] To ensure that your form is working properly kindly uses the form `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html)

File: index.html [hunk 3]
```diff
@@ -371,5 +371,61 @@ <h2>About Myself</h2>
...
+              placeholder="Enter your Email"
+              required
+            />
+          </li>
+          <li>
+            <textarea
+              name="textarea"
+              id="textarea"
+              minlength="50"
+              maxlength="150"
```
Comment (rloterh) [inline index.html]: - Kindly ensure that this textarea has the `maxlength="500"` attribute as specified.

File: index.html [hunk 4]
```diff
@@ -371,5 +371,61 @@ <h2>About Myself</h2>
...
+            <input
+              type="email"
+              name="email_fied"
+              id="email"
+              placeholder="Enter your Email"
+              required
+            />
+          </li>
+          <li>
+            <textarea
```
Comment (rloterh) [inline index.html:410]: - [x] Kindly ensure that this input field has the attribute `required`

File: style.css [hunk 5]
```diff
@@ -215,3 +215,45 @@ section {
   line-height: 20px;
   padding-left: 1.5rem;
 }
+
+/* contact form style */
+
+.form-contain {
```
Comment (rloterh) [inline style.css]: - [x] Your form does not conform to the design as provided in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). We actually want our design to match the provided template design as much as possible.

**Your design:**
image

**The original design:**
image

Kindly ensure to improve on this aspect.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Qeescode, 

You did extremely great on this project!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 

## Good Points 👍  
- Good ReadMe
- No linter errors


## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shaher-11 — APPROVED** [review]
#### Hello @balqees0325,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle::muscle:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Pull request merged**